webapp/accounts/forms.py

```diff
@@ -0,0 +1,39 @@
+from django.contrib.auth.models import User
+from django.contrib.auth.forms import UserCreationForm, UserChangeForm
+from django import forms
+
+
+class RegistrationForm(UserCreationForm):
+    email = forms.EmailField(required=True)
```

Comment:
This had to be manually added despite the `User` model having an `email` field? Thought that'd be added automatically 🤔

Reply:
I thought it was to make the email field a requirement when filling out the form